http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/HttpSamDAO.scala

```diff
@@ -550,7 +550,7 @@ class HttpSamDAO[F[_]](httpClient: Client[F],
     ).use { credential =>
       val scopedCredential = credential.createScoped(saScopes.asJava)
 
-      F.delay(scopedCredential.refresh).map(_ => scopedCredential.getAccessToken)
+      F.blocking(scopedCredential.refresh).map(_ => scopedCredential.getAccessToken)
```

Comment:
Not sure how big a deal this is, but it is essentially a network call to Google.

(`F.blocking` makes it do the task on a `cachedThreadPool`, not the main io-compute worker thread poll)